app/views/bulk_updates/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="row row-content">
+  <section>
+    <h2>
+      Bulk Update <%= @task_id %>
+      <span class="bulk-update-status eui-badge--sm"><%= @task.fetch('task-status') %></span>
+    </h2>
+    <p class="bulk-update-status-message"><%= @task.fetch('status-message', 'Status not retrieved') %></p>
+
+  </section>
+
+  <section>
+    <% unless @task.fetch('collection-statuses', nil).blank? %>
+      <h4>Bulk Update Errors</h4>
+      <table class="space-top space-bot" id="bulk-update-errors-table">
+        <thead>
+          <tr>
+            <th>Entry Title</th>
+            <th>Short Name</th>
+            <th>Version</th>
+            <th>Status Message</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <% @task['collection-statuses'].each do |task_error| %>
```

Comment:
In most spots of this PR you've got a bit far on ensuring we have values to display but on this one you're at risk of throwing a `nil has no method each` due to a lack of default on the 'collection-statuses' key 😃 

When iterating over an array, we should ensure we actually have an array by using `fetch` (most commonly used throughout the app) or even `||`